### PR TITLE
Release: Qubes OS 4.0.4-rc2

### PR DESCRIPTION
**Pull request by andrewdavidwong**
Requested by: QubesOS/qubes-issues#6170

### CODE REVIEW

File: 2021-01-22-qubes-4-0-4-rc2.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+4.0.4.
+
+Qubes OS 4.0.4-rc2 includes many updates over the initial 4.0 release,
+in particular:
+
+- All 4.0 dom0 updates to date
+- Fedora 32 TemplateVM
+- Debian 10 TemplateVM
+- Whonix 15 Gateway and Workstation TemplateVMs
+- Linux kernel 5.4 by default
```
Comment (andrewdavidwong) [inline 2021-01-22-qubes-4-0-4-rc2.md:19]: @marmarek, is it 5.4 *by default*, or just as an option?

File: 2021-01-22-qubes-4-0-4-rc2.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+- Bug fixes from 4.0.4-rc1
```
Comment (andrewdavidwong) [inline 2021-01-22-qubes-4-0-4-rc2.md:20]: @marmarek, we can go into more detail about these if you want, or link to existing issues.

File: 2021-01-22-qubes-4-0-4-rc2.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+If you're willing to [test] this release candidate, you can help to
+improve the stable release by [reporting any bugs you encounter].
+
+
+Release candidate planning
+--------------------------
+
+If no major issues are discovered in 4.0.4-rc2, we expect to announce
+the stable release of 4.0.4 in a couple weeks.
```
Comment (andrewdavidwong) [inline 2021-01-22-qubes-4-0-4-rc2.md:55]: @marmarek, is this still accurate for rc2?

File: 2021-01-22-qubes-4-0-4-rc2.md (hunk 1 above)
Comment (marmarek) [reply 2021-01-22-qubes-4-0-4-rc2.md:19]: Yes, it's the default one.

File: 2021-01-22-qubes-4-0-4-rc2.md (hunk 3 above)
Comment (marmarek) [reply 2021-01-22-qubes-4-0-4-rc2.md:55]: Yes

File: 2021-01-22-qubes-4-0-4-rc2.md (hunk 2 above)
Comment (marmarek) [reply 2021-01-22-qubes-4-0-4-rc2.md:20]: I think this short version is fine.

**Review by marmarek — APPROVED** [review]
```
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

Approved as of b13fea36471946f54785c8829a7835843e6ec9be
-----BEGIN PGP SIGNATURE-----

iQIzBAEBCAAdFiEEnYWYU8Ll1fIX9iss8yiUvpaEk4oFAmAMX08ACgkQ8yiUvpaE
k4qPdA/8CcU5RtQThwsd0N3X4D/uqqRAatLKnfKcclioiNh5/60ZxwKsRcHsFbCY
nMSRuXZeMQJPWT2zvyb2NKxhMRt9TeQR3XTx5NVioCXnv8ksGZqM75zPigX1S3AM
7wMuTeF2GaC05kDAxo0VbX4+zYPUtP42TSBBx7qQnsLMcg8G1DSs/zLc5d2I5A3W
DqiVNBPcF08uRk4hUU8VPZqDNfG6cb69Gwl8DBTHyllB2ypL+B+rtl0Xaljfi/iX
9nCOyIr9D+qbCr+wg/M2ZOX4B9nfFYtF9UyccRDosQ0sWJRhm1/eMUzjIkjhOCEw
F4WAxs9mJrAGlfm5kZWm/o3hROZ36cRoymwvk22zUUd32eyg1uR8UW2hZ103f1zX
gz4nex+nrger5apGPsBFW9ID7Dw2SLsIeceOgSlw/2J1f8Tp5IuA9Mp6PxHdU970
i/ylGr9O5cf92q1DV+5dkJCSiryy/GPBaSPFCbn9u/F8/mNOGVrd2luiTivoVCy9
VDJdiAYVf9W6R2fvwY/jiSi8Q4CZ5qfeR8/W7KtXjIOvUSfq1XeCiuRXGSHkiKK2
hXuc2zQX3H4/h649+aBiLAyfBhrhMsc8QbW7989BOjB/mOEAPxhdaBWVK7MsSMjj
uimK9vGv2Jm5GRY2j33Fr+k6kO7fC+nvhnMShr8cCLbc9YWdbRc=
=ElWY
-----END PGP SIGNATURE-----
```

**Pull request merged**